### PR TITLE
bug 1278498 - cleanup missing_symbols

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by peterbe** [comment]
FYI, I just copied [clean_raw_adi.py](https://github.com/mozilla/socorro/blob/master/socorro/cron/jobs/clean_raw_adi.py) and its test file and modified.

File: socorro/cron/jobs/clean_missing_symbols.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+@with_postgres_transactions()
+@with_single_postgres_transaction()
+class CleanMissingSymbolsCronApp(BaseCronApp):
+    app_name = 'clean-missing-symbols'
+    app_description = """
+    See https://bugzilla.mozilla.org/show_bug.cgi?id=1278498
+
+    By default we truncate to the last 7 days. Normally our general
```
Comment (adngdb) [inline socorro/cron/jobs/clean_missing_symbols.py]: This says 7 days but the config option's default is 5. Which one should it really be?

File: socorro/cron/jobs/clean_missing_symbols.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    retention policy is a lot longer but the missing_symbols
+    is only ever queried for the last day.
+    """
+    required_config = Namespace()
+    required_config.add_option(
+        'days_to_keep',
+        default=5,
+        doc='Number of days of missing symbols to keep in Postgres')
```
Comment (adngdb) [inline socorro/cron/jobs/clean_missing_symbols.py]: nit: closing parenthesis should be on a new line.

File: socorro/cron/jobs/clean_missing_symbols.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    def run(self, connection):
+        cursor = connection.cursor()
+        # Casting to date because stored procs in psql are strongly typed.
```
Comment (adngdb) [inline socorro/cron/jobs/clean_missing_symbols.py]: I don't see what that comment relates to.

**Comment by adngdb** [comment]
r+ with nits.

**Pull request merged**